README.md

```diff
@@ -1,29 +1,60 @@
-# О проекте
-Участие в разработке может принимать любой желающий при условии соблюдения правил разработки. 
+# About project
+The backend for internal company's CRM using Python, Django and GraphQL.
 
-**Приоритет у этой разработки самый низкий! Любая задача или просьба коллеги более важна.**
+Anyone can take part in the development provided abidance with the contribution rules 
 
-### Правила разработки
-* Чтить PEP8 и не нарушать его.
-* Все коменты в коде и названия задач должны быть на английском!
-* Каждый модуль должен быть покрыт тестами не менее 80%.
-* На каждую задачу заводится отдельная ветка.
-* Мёрж в мастер делается через Pull request. Пулреквест невозможен пока не проходят ВСЕ тесты.
+**Pay attention: The priority of this project is the lowest! Any commercial task or colleague's request has more priority.**
```

Comment:
Now it's not true. The priority of the project is medium. The project is very important for the company right now.